File: Stn.java
class Stn 
{
    public static void main(String[] args) 
    {
    int[] arr1 = {2, 2, 7, 5, 3, 3};
    int[] arr2 = {4, 3, 3, 8};

    int n = arr1.length;
    int m = arr2.length;
    int k = Math.max(n,m);
    int[] result = new int[k + 1];
    int carry = 0;
    int i = n-1;
    int j = m-1;

    while(i>=0 || j>=0)
    {
        int sum = carry;
        if(i>=0)
        {
            sum+=arr1[i];
            i--;
        }
        if (j >= 0) 
        {
            sum += arr2[j];
            j--;
        }
        carry = sum/10;
        result[k] = sum % 10;
        k--;
    }

    if (carry != 0) 
    {
        result[k] = carry;
    }
    else
    {
        k++;
    }

    for (int index = k; index < result.length; index++) 
    {
        System.out.print(result[index]);
    }
    }
}

File: stn.py
arr1 = [2, 2, 7, 5, 3, 3]
arr2 = [4, 3, 3, 8]

str1 = ""
str2 = ""

for i in arr1:
    str1+= str(i)

for i in arr2:
    str2+= str(i)

res = int(str1) + int(str2)

output=[]

for i in str(res):
    output+=[int(i)]

print(output)